roles/repository_db/files/usr/local/bin/initialize-repository-db

```diff
@@ -78,8 +78,13 @@ def run_cnxdb_cmd(subcmd, args, extra_args=''):
     `extra_args` is a string of other commandline arguments.
     """
     executable = os.path.join(EXECUTABLE_ENV, 'bin/cnx-db')
-    cmd = "{0} {1} -h {2.host} -p {2.port} -d {2.name} -U {2.user} {3}".format(executable, subcmd, args, extra_args)
-    retcode, out, err = call(cmd)
+    db_url = 'postgres://{}@{}:{}/{}'.format(
+        args.user, args.host, args.port, args.name)
+    db_super_url = 'postgres://postgres@{}:{}/{}'.format(
+        args.host, args.port, args.name)
+    cmd = ' '.join([executable, subcmd, extra_args])
+    retcode, out, err = call(cmd, env={'DB_URL': db_url,
+                                       'DB_SUPER_URL': db_super_url})
```

Comment:
I had completely forgotten about this script. It might be a useful followup to see if this script is actually necessary anymore. IIRC it was put in place, at least partially, to bridge the gap between moving the init scripts out of publishing and archive into cnx-db.

Reply:
Oh... I'll check

Reply:
I removed the initialize-repository-db script :tada: